extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class StaticUserPagesHooks {
+
+	/**
+	 * List of accounts user pages should be rendered using provided i18n strings
+	 */
+	const USER_PAGES_MESSAGES = [
+		Wikia::USER => 'static-user-pages-user-content',
+		'Fandom'    => 'static-user-pages-user-content',
+		'Wikia'     => 'static-user-pages-user-content',
+
+		Wikia::BOT_USER => 'static-user-pages-bot-content',
+		'FandomBot'     => 'static-user-pages-bot-content',
+		'WikiaBot'      => 'static-user-pages-bot-content',
+	];
+
+	/**
+	 * Use StaticUserPagesArticle class to render selected user pages using i18n messages
+	 *
+	 * @param Title $title
+	 * @param $article
+	 * @return bool
+	 */
+	static public function onArticleFromTitle( Title $title, &$article ) : bool {
+		if ( self::isSupportedTitle( $title ) ) {
+			$article = new StaticUserPagesArticle( $title );
+		}
+		return true;
+	}
+
+	/**
+	 * Return i18n message name for a given title (or false if there's none)
+	 *
+	 * @param Title $title
+	 * @return string|false
+	 */
+	static public function getMessageForTitle( Title $title ) {
+		return self::USER_PAGES_MESSAGES[ $title->getText() ] ?: false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)